Source/bmalloc/libpas/src/libpas/pas_mte_config.c

```diff
@@ -142,6 +154,12 @@ static void pas_mte_do_initialization(void)
             *medium_byte = 1;
             *enabled_byte = 1;
             *hardened_byte = 1;
+
+            for (size_t i = 0; i < sizeof(all_bmalloc_runtime_configs) / sizeof(void*); i++) {
+                pas_basic_heap_runtime_config* cfg = all_bmalloc_runtime_configs[i];
+                cfg->base.max_segregated_object_size = PAS_MIN(cfg->base.max_segregated_object_size, 32 * 1024);
+                cfg->base.max_bitfit_object_size = PAS_MIN(cfg->base.max_bitfit_object_size, 32 * 1024);
+            }
```

Comment:
I settled on this approach after going through a few alternatives, and while I do think it's the best way it does bump *all* bmalloc objects of sufficient size to allocate through the large heap -- including those not eligible for delegation.

The alternative that would not do this would be to instead have the segregated-directory slow-refill path check for delegation-eligibility on a per-refill basis, which would allow it to still provide marge bitfit allocations in the case that the directory knows that an allocation would not be eligible for delegation. The biggest downside with that is that it adds significantly more complexity to the question of "where and how is <whatever object> allocated".